package.json

```diff
@@ -47,11 +47,11 @@
   ],
   "license": "MPL-2.0",
   "dependencies": {
-    "ethereumjs-util": "^6.1.0",
-    "level-mem": "^3.0.1",
+    "ethereumjs-util": "^7.0.0",
+    "level-mem": "^5.0.1",
```

Comment:
Would be good if you point this out more prominently somewhere if you do such a high-stake major dependency upgrade.

Reply:
Have changed the PR title on this so it won't be forgotten on the release notes.